### ISSUE TITLE
Login error invalid_token with VK Android app v8.11 and VK Android SDK 

### ISSUE THREAD

**Issue by yishigong** [VKCOM/vk-android-sdk#524]
Hi VK team!

App integrated with VK Android SDK v1.6.9, device has VK Android app v7.35 installed. 
When redirected to VK app for authorization, the below error is shown on a VK-hosted page:

{"error":"invalid_token","error_description":"token is incorrect"}

![image](https://user-images.githubusercontent.com/43773257/210061077-0f1a9067-5f6a-40cf-82dd-5a8933262d84.png)

Kindly help verify if this is a known issue, and if it is fixed in any newer SDK versions. 
If not, any estimation on fixing it? 

Thank you!

**Comment by yishigong** [comment]
Also reproducible occasionally with Android VK app v8.11 installed on device.

**Comment by 7va** [comment]
Still there

**Comment by 1352101891** [comment]
Yes, I also encountered the same problem. my android VK app version is v8.7.2.  vk-android-sdk is v4.1.0, Occasionally appear this problem
